### PR TITLE
Allow uploading .jpeg files on Linux

### PR DESCRIPTION
**Pull request by cwarnermm**
In-product text update for: https://github.com/mattermost/mattermost-webapp/pull/8224

Updated profile picture upload text to clarify support for JPEG image files.

### CODE REVIEW

**Comment by cwarnermm** [comment]
@esethna - Who from the mobile side should review this in-product text update related to https://github.com/mattermost/mattermost-webapp/pull/8224 (allow users to upload .jpeg files as profile pictures)?

**Review by esethna — APPROVED** [review]
Looks fine, we can merge

**Comment by esethna** [comment]
I think this also needs to be changed in the file itself, not just the translation file (as per my example PR I shared with you) @cwarnermm 

Here if I'm not mistaken (I just search for the string in the webapp repo and go to the file where the string is used)
https://github.com/mattermost/mattermost-webapp/blob/74cbcb7c3f0dff7e21ecb88776b8841dd1fa55e6/components/user_settings/general/user_settings_general.tsx#L1263

**Comment by cwarnermm** [comment]
@koox00 - I see that a release note is needed for this PR. Could you teach me how to add a release note for this in-product text change so that I can unblock this PR? Thanks so much!

**Review by koox00 — APPROVED** [review]

**Pull request merged**
